core/src/main/scala/zio/logging/Logger.scala

```diff
@@ -0,0 +1,41 @@
+package zio.logging
+
+import zio.internal.Tracing
+import zio.internal.stacktracer.Tracer
+import zio.internal.stacktracer.ZTraceElement.{ NoLocation, SourceLocation }
+import zio.internal.stacktracer.impl.AkkaLineNumbersTracer
+import zio.internal.tracing.TracingConfig
+import zio.{ FiberRef, UIO, URIO, ZIO }
+
+/**
+ * A logger of strings.
+ */
+trait Logger[-R] extends LoggerLike[String, R]
```

Comment:
Let's delete the `R` parameter from `Logger` as it won't be supported in RC18 and later.

Reply:
I added to avoid usage of ```provide``` in ```make[R]``` but I agree it was silly reason.